terraform/file-hosting/vcl/main.vcl

```diff
@@ -61,6 +69,18 @@ sub vcl_recv {
         # access the files stored there.
         set req.http.Authorization = "AWS " var.AWS-Access-Key-ID ":" digest.hmac_sha1_base64(var.AWS-Secret-Access-Key, "GET" LF LF LF req.http.Date LF "/" var.S3-Bucket-Name req.url.path);
     }
+    # If our file request is being dispatched to GCS, setup the request to correctly
+    # access GCS and authorize ourselves with GCS interoperability credentials.
+    if (req.backend == GCS && req.url ~ "^/packages/[a-f0-9]{2}/[a-f0-9]{2}/[a-f0-9]{60}/") {
+        # Setup our environment to better match what S3 expects/needs
```

Comment:
Comment references S3

Reply:
Thanks, fixed in e0f4cbe9276ac159711f8f95a9711a03ad19b92a